core/src/main/java/org/apache/accumulo/core/rpc/TTimeoutTransport.java

```diff
@@ -100,6 +103,12 @@ TTransport createInternal(SocketAddress addr, long timeoutMillis) throws TTransp
       return new TIOStreamTransport(input, output);
     } catch (IOException e) {
       closeSocket(socket, e);
```

Comment:
Can this closeSocket trigger its own IOException?

Reply:
Uh... that snippet does not look nice... not sure how to format that nicely: but it's in TTimeoutTransport.java

Reply:
@nikita-sirohi I formatted your code snippet. See https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting